cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -197,7 +197,7 @@ def update_single_sample_read_count(self, sample_id: str) -> None:
                 sample_internal_id=sample_id
             )
             LOG.info(f"Updating sample {sample_id} with read count {sample_read_count}")
-            sample.reads = sample_read_count
+            sample.calculated_read_count = sample_read_count
```

Comment:
So what happens to the reads column in this case?

Reply:
Nothing, it is updated with the CGstats logic that we aim to deprecate. After some reviewing of the code there are some more connections that we need to sever one by one to have this all go away. So a parallel approach in this case might be healthier.